framework/flamingo/sessions.go

```diff
@@ -124,9 +127,18 @@ func (m *SessionModule) setSessionstoreOptions(options *sessions.Options) {
 	options.Domain = ""
 	options.Path = m.path
 	options.MaxAge = m.maxAge
-	options.Secure = m.secure
+	options.Secure = true
```

Comment:
why do we need this to be always true?

Reply:
@tessig  yup, you are right, it was pushed by mistake.